quantum/quantum.h

```diff
@@ -73,6 +73,9 @@ extern uint32_t default_layer_state;
   #ifdef AUDIO_CLICKY
     #include "process_clicky.h"
   #endif // AUDIO_CLICKY
+  #ifdef DEFAULT_LAYER_SONGS
+    extern float default_layer_songs[][16][2];
```

Comment:
Wouldn't this make more sense to add to your keymap.c file, than here? 

Or was there an issue with doing so?

Reply:
Ah! The issue was: my C skills are rusty. :sweat_smile:

Your suggestion works fine, and I learned more about what `extern` does. I'll include the changes for this and your other suggestions in my next push.

Thank you very much for reviewing this!